Auctionator.lua

```diff
@@ -767,9 +767,24 @@ function Atr_InitScanDB()
     AUCTIONATOR_PRICE_DATABASE["__dbversion"] = 4;
   end
 
+  if (AUCTIONATOR_PRICE_DATABASE and AUCTIONATOR_PRICE_DATABASE["__dbversion"] == 4) then
+    for realm_fac, data in pairs (AUCTIONATOR_PRICE_DATABASE) do
+        zc.md ("migrating Auctionator db to version 5 for:", realm_fac);
+		if realm_fac:find("_Alliance") ~= nil then
```

Comment:
Nitpick: can you replace tabs with 2 spaces, and ensure indentation is consistently two spaces? Yes, I'm one of *those* programmers.